examples/rllib/utils.py

```diff
@@ -13,13 +13,22 @@
 # limitations under the License.
 """MeltingPotEnv as a MultiAgentEnv wrapper to interface with RLLib."""
 
+from typing import Tuple
+
 import dm_env
 import dmlab2d
 from gym import spaces
+from ml_collections import config_dict
 import numpy as np
+from ray.rllib.agents import trainer
 from ray.rllib.env import multi_agent_env
+from ray.rllib.policy.sample_batch import DEFAULT_POLICY_ID
 import tree
 
+from meltingpot.python import substrate
+from meltingpot.python.bot import Policy
```

Comment:
Please don't import module objects directly. Instead, import the module and explicitly refer to the objects later, from the module. E.g.:

```
from meltingpot.python import bot
...
class RayModelPolicy(bot.Policy):
...
```

Reply:
Ok